### PR TITLE
multiple api version

### PR DESCRIPTION
**Pull request by jianyexi**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 0.7.2
+
+- Add rule v to validate if the default tag in readme.md contains multiple API version.
```
Comment (ruowan) [inline CHANGELOG.md:5]: what's is v?

File: src/index.ts [hunk 2]
```diff
@@ -90,6 +97,73 @@ const isAutoRestMd = (m: md.MarkDownEx) =>
...
+    return undefined
+  }
+}
+
+export const getDefaultTag = (markDown: commonmark.Node): string => {
+  const startNode = markDown
+  const codeBlockMap = openApiMd.getCodeBlocksAndHeadings(startNode)
+
+  const latestHeader = 'Basic Information'
+  const lh = codeBlockMap[latestHeader]
```
Comment (ruowan) [inline src/index.ts:113]: more readable naming.

File: src/index.ts [hunk 3]
```diff
@@ -90,6 +97,73 @@ const isAutoRestMd = (m: md.MarkDownEx) =>
...
+    const block = codeBlockMap[idx]
+    if (!block || !block.info || !block.literal || !/^(yaml|json)$/.test(block.info.trim().toLowerCase())) {
+      continue
+    }
+    const latestDefinition = safeLoad(block.literal)
+    if (latestDefinition && latestDefinition.tag) {
+      return latestDefinition.tag
+    }
+  }
+  return ''
```
Comment (ruowan) [inline src/index.ts:130]: Why return an empty string? If not found, return undefined

Comment (jianyexi) [reply src/index.ts:130]: This case, it's fine to use empty string to indicate not found

**Review by ruowan — APPROVED** [review]

**Review by ruowan — APPROVED** [review]

**Pull request merged**